lib/tests/streamlit/elements/plotly_chart_test.py

```diff
@@ -94,6 +95,47 @@ def test_st_plotly_chart_use_container_width_true(self):
         self.assertNotEqual(el.plotly_chart.config, "")
         self.assertEqual(el.plotly_chart.use_container_width, True)
 
+    def test_works_with_element_replay(self):
+        """Test that it works with element replay if used as non-widget element."""
```

Comment:
nit: """Test that element replay works for plotly if used as non-widget element"""

Reply:
Updated